admin/app/views/football/main.scala.html

```diff
@@ -4,6 +4,7 @@
 @import controllers.admin.routes.UncachedWebAssets
 @import play.api.Mode.Dev
 @import conf.Static
+@import html.HtmlPageHelpers.pillarCardCSSFileContent
```

Comment:
@TBonnin this import doens't work but im not sure why – guess it's not available to templates. any idea how to accomplish this?

same with onward/app/views/richLink.scala.html

Reply:
Looks like it works. What error are you seeing?

Reply:
hmm weird, it was erroring running `compile` last night, but seems ok now. maybe your reply was enough to fix it!